common/theme.c

```diff
@@ -22,9 +36,11 @@ void themeStartup(ThemePreset preset) {
         .backgroundColor = MakeColor(233, 236, 241, 255),
         .highlightColor = MakeColor(91, 237, 224, 255),
         .separatorColor = MakeColor(219, 218, 219, 255),
+        .borderColor = MakeColor(255,255,255,255),
+        .borderTextColor = MakeColor(64,64,64,255),
         .enableWaveBlending = 0,
-        .buttonAText = "\uE0E0",
-        .buttonBText = "\uE0E1",
+        .buttonAText = buttonAText,/*setting the buttonAText = "\uE0E0" directly allocates the literal on the stack */
```

Comment:
The claim in this comment is false.

Reply:
hmmmm, then what exactly does it do? when i googled it this was the conclusion that i came too... and i talked to my other coworkers about this as well and they thought the same thing.